kinisi/parser.py

```diff
@@ -225,3 +225,129 @@ def get_indices(
+
+        self.distance_unit = distance_unit
+
+        structure, coords, latt = self.get_structure_coords_latt(universe, progress, sub_sample_atoms, sub_sample_traj)
+
+        indices, drift_indices = self.get_indices(structure, specie)
+
+        super().__init__(coords, latt, indices, drift_indices, time_step, step_skip, dt, dimension)
+
+    def get_structure_coords_latt(
+        self,
+        universe: 'MDAnalysis.core.universe.Universe',
+        progress: bool = True,
+        sub_sample_atoms: int = 1,
+        sub_sample_traj: int = 1,
+    ) -> Tuple["MDAnalysis.core.universe.Universe", sc.Variable, sc.Variable]:
+        """
+        Obtain the initial structure, coordinates, and lattice parameters from an MDAnalysis.Universe object.
+
+        :param universe: MDanalysis universe object.
+        :param progress: Whether to show a progress bar when reading in the structures.
+        :param sub_sample_atoms: Subsample the atoms in the trajectory. Optional, defaults to 1.
+        :param sub_sample_traj: Subsample the trajectory. Optional, defaults to 1.
+
+        :returns: A tuple of:  the initial structure (as
+            a :py:class:`MDAnalysis.core.universe.Universe`), coordinates (as
+            a :py:mod:`scipp` array with dimensions of `time`, `atom`, and `dimension`),
+            and lattice parameters (as a :py:mod:`scipp` array with dimensions `time`,
+            `dimension1`, and `dimension2`).
+        """
+        first = True
+        coords_l = []
+        latt_l = []
+        if progress:
+            iterator = tqdm(universe.trajectory[::sub_sample_traj], desc='Reading Trajectory')
+        else:
+            iterator = universe.trajectory[::sub_sample_traj]
+
+        for struct in iterator:
+            if first:
+                structure = universe.atoms[::sub_sample_atoms]
+                first = False
+            matrix = np.array(struct.triclinic_dimensions)
+            inv_matrix = np.linalg.inv(matrix)
+            coords_l.append(np.dot(universe.atoms[::sub_sample_atoms].positions, inv_matrix))
+            latt_l.append(np.array(matrix))
+
+        coords_l = np.array(coords_l)
+        latt_l = np.array(latt_l)
+
+        coords = sc.array(dims=['time', 'atom', 'dimension'], values=coords_l, unit=sc.units.dimensionless)
+        latt = sc.array(dims=['time', 'dimension1', 'dimension2'], values=latt_l, unit=self.distance_unit)
+
+        return structure, coords, latt
+
+    def get_indices(
+        self,
+        structure: "MDAnalysis.universe.Universe",
+        specie: str,
+    ) -> Tuple[sc.Variable, sc.Variable]:
+        """
+        Determine framework and non-framework indices for an :py:mod:`MDAnalysis` compatible file.
+
+        :param structure: Initial structure.
+        :param specie: Specie to calculate diffusivity for as a String, e.g. :py:attr:`'Li'`.
+
+        :return: Tuple containing: indices for the atoms in the trajectory used in the calculation of the
```

Comment:
Having a second colon in the line might break the automatic documentation generation. 
```suggestion
        :return: Tuple containing indices for the atoms in the trajectory used in the calculation of the
```